core/domain/config_domain.py

```diff
@@ -297,6 +299,11 @@ def normalize(self, value):
         Returns:
             instance. The normalized object.
         """
+        string_types = (python_utils.BASESTRING, python_utils.UNICODE)
```

Comment:
What about testing things that can also contain strings (e.g. lists & dicts)?

Reply:
Ah, yes, totally omitted that, I have refactored it.